src/uwloc/data/main.py

```diff
@@ -44,15 +49,15 @@ def import_wavs(
     files = sorted(glob.iglob(os.path.join(wavs_dir, WAV_PATTERN), recursive=True))
     for f in files:
         try:
-            (device, timestamp, rate, data) = read_wav(f)
+            device, timestamp, rate, data = read_wav(f)
             if len(device) == 0:
-                typer.echo(f"{f} is missing the Device ID, skipping")
+                logger.warning(f"{f} is missing the Device ID, skipping")
                 continue
             secs = data.size // rate
             typer.echo(f"Importing  {f}. Timestamp: {timestamp} Device: {device}, secs: {secs}")
             write_row(dbpath, device, timestamp, data)
         except Exception as e:
-            typer.echo(f"Warning: Failed to import {f}. {e}", err=True)
+            logger.warning(f"Warning: Failed to import {f}. {e}")
```

Comment:
Probably better to log this as `ERROR` level: https://docs.python.org/3/library/logging.html#logging.Logger.error